modules/sdk-coin-ethw/test/lib/recovery-nocks.ts

```diff
@@ -1,265 +1,172 @@
 import nock from 'nock';
 
-import { Environments } from '@bitgo/sdk-core';
 import { TestBitGoAPI } from '@bitgo/sdk-test';
 
 const nockEthData: any[] = [
   {
     params: {
-      module: 'account',
-      action: 'txlist',
-      address: '0x74c2137d54b0fc9f907e13f14e0dd18485fee924',
+      method: 'eth_getTransactionCount',
```

Comment:
if this is a breaking API changes, the commit message needs to add `BREAKING` . See: https://github.com/BitGo/BitGoJS/pull/2565

Reply:
this is non API breaking, ethw is now using ethereumpow RPC instead of etherscan